Application/Queries/GetAllBooksQuery.cs

```diff
@@ -18,6 +18,7 @@ public async Task<List<Book>> GetAllAsync(long tenantId)
       .Books
```

Comment:
Let's add AsNoTracking() here for read queries, it looks like it was forgotten to be added.